bijection-core/src/main/scala/com/twitter/bijection/StringInjections.scala

```diff
@@ -53,7 +53,7 @@ trait StringInjections extends NumericInjections {
           val decBuf = decRef.get
           val dec = decBuf._1
           val buf = decBuf._2
-          val maxSpace = (b.length * dec.maxCharsPerByte).toInt + 1
+          val maxSpace = (b.length * (dec.maxCharsPerByte.toDouble)).toInt + 1
```

Comment:
I'd put a comment, that for cases where it's possible to have 2B, just try using 2B.  I'd also drop the () and maybe just toDouble on the length, since that's the thing that can be large.